src/components/EmojiPicker/EmojiPicker.tsx

```diff
@@ -45,34 +48,39 @@ const EmojiPicker = forwardRef((props, ref) => {
      *
      * Don't directly get the ref from emojiPopoverAnchorRef, instead use getEmojiPopoverAnchor()
      */
-    const getEmojiPopoverAnchor = useCallback(() => emojiPopoverAnchorRef.current || emojiPopoverAnchorRef, []);
+    const getEmojiPopoverAnchor = useCallback(() => emojiPopoverAnchorRef.current ?? emojiPopoverAnchorRef, []);
 
     /**
      * Show the emoji picker menu.
      *
-     * @param {Function} [onModalHideValue=() => {}] - Run a callback when Modal hides.
-     * @param {Function} [onEmojiSelectedValue=() => {}] - Run a callback when Emoji selected.
-     * @param {React.MutableRefObject} emojiPopoverAnchorValue - Element to which Popover is anchored
-     * @param {Object} [anchorOrigin=DEFAULT_ANCHOR_ORIGIN] - Anchor origin for Popover
-     * @param {Function} [onWillShow] - Run a callback when Popover will show
-     * @param {String} id - Unique id for EmojiPicker
-     * @param {String} activeEmojiValue - Selected emoji to be highlighted
+     * @param [onModalHideValue=() => {}] - Run a callback when Modal hides.
+     * @param [onEmojiSelectedValue=() => {}] - Run a callback when Emoji selected.
+     * @param emojiPopoverAnchorValue - Element to which Popover is anchored
+     * @param [anchorOrigin=DEFAULT_ANCHOR_ORIGIN] - Anchor origin for Popover
+     * @param [onWillShow] - Run a callback when Popover will show
+     * @param id - Unique id for EmojiPicker
+     * @param activeEmojiValue - Selected emoji to be highlighted
      */
-    const showEmojiPicker = (onModalHideValue, onEmojiSelectedValue, emojiPopoverAnchorValue, anchorOrigin, onWillShow, id, activeEmojiValue) => {
+    const showEmojiPicker = (
+        onModalHideValue: OnModalHideValue,
+        onEmojiSelectedValue: OnEmojiSelected,
+        emojiPopoverAnchorValue: EmojiPopoverAnchor,
+        anchorOrigin?: AnchorOrigin,
+        onWillShow?: OnWillShowPicker,
+        id?: string,
+        activeEmojiValue?: string,
+    ) => {
         onModalHide.current = onModalHideValue;
         onEmojiSelected.current = onEmojiSelectedValue;
         activeEmoji.current = activeEmojiValue;
         emojiPopoverAnchorRef.current = emojiPopoverAnchorValue;
         const emojiPopoverAnchor = getEmojiPopoverAnchor();
-        if (emojiPopoverAnchor.current && emojiPopoverAnchor.current.blur) {
-            // Drop focus to avoid blue focus ring.
-            emojiPopoverAnchor.current.blur();
-        }
+        // Drop focus to avoid blue focus ring.
+        (emojiPopoverAnchor?.current as HTMLDivElement)?.blur();
 
-        const anchorOriginValue = anchorOrigin || DEFAULT_ANCHOR_ORIGIN;
+        const anchorOriginValue = anchorOrigin ?? DEFAULT_ANCHOR_ORIGIN;
 
-        calculateAnchorPosition(emojiPopoverAnchor.current, anchorOriginValue).then((value) => {
-            // eslint-disable-next-line es/no-optional-chaining
+        calculateAnchorPosition(emojiPopoverAnchor.current as View, anchorOriginValue).then((value) => {
```

Comment:
Let's remove this assertion:
```suggestion
        calculateAnchorPosition(emojiPopoverAnchor?.current, anchorOriginValue).then((value) => {
```

This requires a change in ContextMenuAnchor:
```ts
type ContextMenuAnchor = View | RNText | TextInput | HTMLDivElement | null | undefined;
```